webpack.config.js

```diff
@@ -66,6 +66,7 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
                     'vendor/friendsofsymfony/jsrouting-bundle/Resources/public/js'
                 ),
             },
+            symlinks: false,
             modules: ['node_modules', nodeModulesPath],
```

Comment:
we can remove this if we enable disable `symlinks` because the `assets/admin` folder is a parent of the symlinked location 🙂